### PR TITLE
[CI] enable github actions

### PR DESCRIPTION
**Pull request by wtatarski**

### CODE REVIEW

**Comment by mithro** [comment]
@wtatarski - Looks like these are failing on your repository at https://github.com/antmicro/verible/actions/runs/362003112 ?

File: .github/workflows/script.sh [hunk 1]
```diff
@@ -15,28 +15,21 @@
...
 case "$MODE" in
-test)
-    bazel test -c opt "${BAZEL_OPTS[@]}" "${BAZEL_CXXOPTS[@]}" //...
-    ;;
-
-compile)
-    bazel build -c opt "${BAZEL_OPTS[@]}" "${BAZEL_CXXOPTS[@]}" //...
+compile-n-test)
+    bazel test --noshow_progress "${BAZEL_CXXOPTS[@]}" //...
+    bazel build -c opt --noshow_progress "${BAZEL_CXXOPTS[@]}" //...
```
Comment (fangism) [inline .github/workflows/script.sh:27]: Both should use `-c opt` consistently so the two commands share the cache.  (Reminder: `CHECK`s in our code are always-on unaffected by `NDEBUG`)

BTW, I'm not opposed running a duplicate job without `-c opt` (or other configuration options) once we get the first one set up.

File: .github/workflows/success.sh [hunk 2]
```diff
@@ -15,25 +15,21 @@
...
     # Set up things for GitHub Pages deployment
-    ./.github/travis/github-pages-setup.sh
+    ./.github/workflows/github-pages-setup.sh
     ;;
 
 bin)
     # Create a tag of form v0.0-183-gdf2b162-20191112132344
     rm -rf Docker/out
-    git tag $TRAVIS_TAG || true
+    git tag $TAG || true
```
Comment (fangism) [inline .github/workflows/success.sh:32]: general: "quote-protect" variable uses, so `"$TAG"`

File: Docker/docker-run.sh [hunk 3]
```diff
@@ -17,12 +17,12 @@ set -x
...
 
 DIRS=${1:-$(find -mindepth 1 -maxdepth 1 -type d)}
-export TRAVIS_TAG=${TRAVIS_TAG:-$(git describe --match=v*)}
+export TAG=${TAG:-$(git rev-parse --short "$GITHUB_SHA")}
 
 ./docker-generate.sh $DIRS
 
 for DIR in $DIRS; do
-    IMAGE=verible:$DIR-$TRAVIS_TAG
+    IMAGE=verible:$DIR-$TAG
```
Comment (fangism) [inline Docker/docker-run.sh:25]: Recommend: `"verible:$DIR-$TAG"`

**Review by fangism — CHANGES_REQUESTED** [review]
Mostly looks good to me, but I have less experience setting these up.

**Review by fangism — APPROVED** [review]

**Comment by wtatarski** [comment]
@mithro  These tests fails also on other PRs so this is not the issue for this PR.

**Comment by wtatarski** [comment]
Should be OK now. I introduce some small fixes. Errors in checks are related to new commits on master - I did a rebase.

**Comment by fangism** [comment]
> Should be OK now. I introduce some small fixes. Errors in checks are related to new commits on master - I did a rebase.

Fix coming.

**Comment by fangism** [comment]
> > Should be OK now. I introduce some small fixes. Errors in checks are related to new commits on master - I did a rebase.
> 
> Fix coming.

Can you sync to top-of-tree one more time?

**Comment by wtatarski** [comment]
Looks like it is OK now. Here are github actions tests https://github.com/antmicro/verible/actions/runs/367735747

**Comment by fangism** [comment]
> Looks like it is OK now. Here are github actions tests https://github.com/antmicro/verible/actions/runs/367735747

Great progress, let's pull this one in immediately.

**Comment by fangism** [comment]
> > Looks like it is OK now. Here are github actions tests https://github.com/antmicro/verible/actions/runs/367735747
> 
> Great progress, let's pull this one in immediately.

The other io_kythe failure looks like https://github.com/google/verible/issues/584, so that might be the next thing.

**Comment by wtatarski** [comment]
@fangism  so you want me to take a look on #584? Becasue I pick #350 earlier, but if this is more important I can take a look at this.

**Comment by fangism** [comment]
> @fangism so you want me to take a look on #584? Becasue I pick #350 earlier, but if this is more important I can take a look at this.

#584 is kind of blocking releases at the moment, so probably more important.  I commented in #584 that it might be simple a matter of installing `git` on the executor.  Could you try that and see if the builds get farther along?  (And let us know how far you get.). Thanks again for your help.

**Pull request closed without merging**